controllers/auth.go

```diff
@@ -59,6 +59,11 @@ func AuthenticateHandler(c echo.Context) error {
 	}
 
 	if u.Username == username && u.ValidPassword(password) {
```

Comment:
As we're introducing a new conditional inside another conditional we could do a small refactor on this method so it becomes on a conditional -> response, conditional -> response ...

Example:
```go
if !(u.Username == username && u.ValidPassword(password)) {
    return echo.NewHTTPError(403, "The keypair user / password does not match any user on the database, please try again")
}
...

if err := h.ValidCliVersion(c.Request()); err != nil {
     return echo.NewHTTPError(403, err.Error())
}

return c.JSON(http.StatusOK, map[string]string{"token": t})
```

So the code is more readable